### ISSUE TITLE
Disable line truncation

### ISSUE THREAD

**Issue by OrangeDog** [kivy/buildozer#1387]
All output appears to be truncated to the terminal width and replaced with e.g. `...(and 132 more)`
This causes vital debugging information to be lost.

It should be possible to disable this and allow the terminal to wrap itself, especially if the output is to a pipe or file.

**Comment by OrangeDog** [comment]
@HyTurtle yes, that's the log level. That has no effect on the horizontal truncation.

**Comment by HyTurtle** [comment]
If you're sure - then would be a p4a issue rather than a buildozer one, https://github.com/kivy/python-for-android/blob/8336cea09c572c2a234ddda211226a1eb8b2ad1d/pythonforandroid/logger.py#L156 it shouldn't be shortened if log level is higher than debug

**Comment by OrangeDog** [comment]
There is no level higher than debug...

**Comment by HyTurtle** [comment]
Info is higher than debug (in logging hierarchy); that does seem to be a bug though - if that is where the shortening is coming from that is - misspoke a touch it shouldn't shorten if its set as debug (which probably shouldn't be the default), do you get less output setting that to info or the same?

Regardless would probably need to move to the p4a repo and can you check which branch you are using?

**Comment by OrangeDog** [comment]
Setting it to info (`log_level = 1`) gives significantly less output.

**Comment by OrangeDog** [comment]
There appears to be a recent fix in p4a. Setting `p4a.branch = develop` removes the truncation.
